contracts/modules/TransferManager.sol

```diff
@@ -52,12 +52,16 @@ contract TransferManager is OnlyOwnerModule, BaseTransfer, LimitManager {
     uint256 public securityPeriod;
     // The execution window
     uint256 public securityWindow;
+    // The default limit
+    uint128 public defaultLimit;
     // The Token storage
     ITransferStorage public transferStorage;
     // The Token price provider
     TokenPriceProvider public priceProvider;
     // The previous limit manager needed to migrate the limits
-    LimitManager public oldLimitManager;
+    TransferManager public oldLimitManager;
```

Comment:
For clarity, since we are doing away with the concept of a LimitManager, I would rename that variable `oldTransferManager`